README.md

```diff
@@ -3,7 +3,7 @@
 [![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](http://standardjs.com/)
 [![Build Status](https://travis-ci.org/fastify/fastify-plugin.svg?branch=master)](https://travis-ci.org/fastify/fastify-plugin)
 
-`fastify-plugin` is a plugin helper for [Fastify](https://github.com/fastify/fastify).  
+`fastify-plugin` is a plugin helper for [Fastify](https://github.com/fastify/fastify).
```

Comment:
Can you add a `<br/>` at the end of this line?

Reply:
@delvedor Done. 👍